data/sql/headers.go

```diff
@@ -121,10 +121,20 @@ const (
     `
 
 	sqlSelectTips = `
-	SELECT hash, height, version, merkleroot, nonce, bits, chainwork, previousblock, timestamp, cumulatedWork, header_state
-	FROM headers
-	WHERE hash NOT IN (SELECT previousblock
-					   FROM headers)
+	with mainTip as (
+	select hash, height, version, merkleroot, nonce, bits, chainwork, previousblock, timestamp, header_state, cumulatedWork
+	from headers
+	where header_state = 'LONGEST_CHAIN'
+	order by height
```

Comment:
It doesn't work, so I assume what's happening is this should be DESC order - I'll try

Reply:
Confirmed - approved.